src/Microsoft.AspNet.Identity/UserManager.cs

```diff
@@ -1838,42 +1994,8 @@ public virtual Task<IList<TUser>> GetUsersInRoleAsync(string roleName)
             return store.GetUsersInRoleAsync(roleName, CancellationToken);
         }
 
-        /// <summary>
-        ///     Logs the current Identity Result and returns result object
-        /// </summary>
-        /// <param name="result"></param>
-        /// <param name="user"></param>
-        /// <param name="methodName"></param>
-        /// <returns></returns>
-        protected async Task<IdentityResult> LogResultAsync(IdentityResult result,
-            TUser user, [System.Runtime.CompilerServices.CallerMemberName] string methodName = "")
-        {
-            result.Log(Logger, Resources.FormatLoggingResultMessage(methodName, await GetUserIdAsync(user)));
-            return result;
-        }
-
-        /// <summary>
-        ///     Logs result of operation being true/false
-        /// </summary>
-        /// <param name="result"></param>
-        /// <param name="user"></param>
-        /// <param name="methodName"></param>
-        /// <returns>result</returns>
-        protected async Task<bool> LogResultAsync(bool result,
-            TUser user, [System.Runtime.CompilerServices.CallerMemberName] string methodName = "")
-        {
-            var baseMessage = Resources.FormatLoggingResultMessage(methodName, await GetUserIdAsync(user));
-            if (result)
-            {
-                Logger.LogInformation(string.Format("{0} : {1}", baseMessage, result.ToString()));
-            }
-            else
-            {
-                Logger.LogWarning(string.Format("{0} : {1}", baseMessage, result.ToString()));
-            }
-
-            return result;
-        }
+        protected virtual async Task<string> GetScopeMessageforUserAsync(TUser user, [CallerMemberName] string methodName = null)
```

Comment:
Correct PascalCase name would be `GetScopeMessageForUserAsync` (with an uppercase 'F'). My preference would be `GetLoggingScopeMessageAsync`.